text/0010-gatsby-docs-localization

```diff
@@ -0,0 +1,104 @@
+* Start Date: 2019-09-11
+* RFC PR:
+* Gatsby Issue:
+
+## Summary
+
+Begin the process of localizing the documentation of Gatsby into non-English languages.
+
+## Motivation
+
+[You Belong Here.](https://www.gatsbyjs.org/blog/2018-09-07-gatsby-values/#you-belong-here)
+
+Gatsby believces that everyone belongs in Open Source, and that includes developers that do not speak English. Having access to translated documentation opens up the ability to create websites to people who don't speak English and don't have the privileges to be able to learn.
+
+## Detailed Design
+
+There are two main approaches we can take for the translations. Whichever path we choose depends on the prorities of the Gatsby team, hence the Request for Comments. The options are: (1) doing translations on GitHub directly or (2) using a translation SaaS platform such as Crowdin. This document will go over both alternatives and the advantages and drawbacks of each option.
+
+## Alternative 1: GitHub-based translations
+
+The first and simplest option is to simply create and host the translations on GitHub itself, along with the rest of the Gatsby documentation. For smaller websites (like [Dan Abramov's blog](https://github.com/gaearon/overreacted.io)) it's enough to put each translation in a markdown file next to the source file and use `gatsby-i18n`.
+
+For larger websites, especially documentation websites that have lots of PRs and require versioning and up-to-date content, this isn't enough. The docs for [Angular](https://github.com/angular/angular-cn/tree/aio), [Vue](https://github.com/vuejs/jp.vuejs.org), and [React](https://github.com/reactjs/es.reactjs.org) store their translations in copies of the repo.
+
+### Advantages
+
+This was the approach that we took for translating the React docs. It appealed to us for a lot of reasons:
+
+* Open Source Contributors tend to be familiar with Git
+* No need to integrate with an external API and software (aside from GitHub)
+* People want to be listed as contributors of the React/Gatsby organization on GitHub, and it's an incentive to contribute.
+* Can use GitHub features such as Code Owners and pull requests as quality assurance
+
+### Drawbacks
+
+The main drawback of this approach is that the Gatsby docs currently live in a monorepo with the rest of the Gatsby source, which means we can't just do a copy of the entire website and use `git merge` like we do with React (and Vue).
+
+This is also an issue if we want translations to live under a path (e.g. `gatsbyjs.org/ja` rather than `ja.gatsbyjs.org`).
```

Comment:
For this, we could clone each language repo for doing the build.

Reply:
It'd be a lot simpler to have one site & a better experience as people could easily jump between translations e.g. with a dropdown.

Reply:
the best way to encourage others to write translations is to make it simpler to write them. I'm translating the docs (tutorials) to [spanish myself](https://github.com/horacioh/gatsby/tree/translation/spanish)

One thing about the language dropdown: because the docs are constantly changing and hopefully a lot of people would help translating them, if a web user changes the dropdown language and that page has not being translate yet, It's a little harder to handle those cases gracefully.

Another way I suggest is how Dan Abramov does it on his blog:
<img width="853" alt="Screen Shot 2019-09-14 at 1 31 02 AM" src="https://user-images.githubusercontent.com/725120/64900137-6edaa100-d68f-11e9-8f2c-6ce78ce31578.png">

one drawback of this approach is that is harder for a web user to change the overall language.

hope this helps!

Reply:
We could do what wikipedia does: only list languages in the dropdown that have the current page translated. You could have a "More Languages" link at the bottom to take you to a directory for all the languages.

We'd love your help on the Spanish documentation!

Reply:
which does the trick and looks real nice and simple.